### PR TITLE
Konyang: PACHROM Water Barge

### PR DESCRIPTION
**Pull request by RustingWithYou**
Adds a PACHROM corporation water transport barge which can spawn in the Haneunim sector, with ghostroles for the crew.

image

### CODE REVIEW

**Comment by RustingWithYou** [comment]
!review

File: maps/away/ships/konyang/water_barge/water_barge.dm [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+		"water_barge_nav1",
+		"water_barge_nav2",
+		"water_barge_nav3",
+		"water_barge_nav4",
+		"water_barge_dock",
+		"water_barge_dock_s"
+	)
+
+/obj/effect/overmap/visitable/ship/water_barge/New()
+	designation = "[pick("THERES WATER ON HERE BRO")]"
```
Comment (DreamySkrell) [inline maps/away/ships/konyang/water_barge/water_barge.dm:43]: .

**Review by DreamySkrell — APPROVED** [review]
- fix this
- same comment on grates as on the air konyang vessel
- otherwise looks fine I think

**Comment by DreamySkrell** [comment]
image
this is failing the checks, probably a typo

**Review by Alberyk — APPROVED** [review]

**Pull request merged**